packages/bloc/lib/src/bloc.dart

```diff
@@ -76,16 +75,24 @@ abstract class Bloc<Event, State> extends Stream<State> {
 
   /// Closes the [Event] and [State] [Stream]s.
   /// This method should be called when a [Bloc] is no longer needed.
-  /// Once `dispose` is called, events that are `dispatched` will not be
+  /// Once `close` is called, events that are `added` will not be
   /// processed and will result in an error being passed to `onError`.
-  /// In addition, if `dispose` is called while [Event]s are still being processed,
+  /// In addition, if `close` is called while [Event]s are still being processed,
   /// any [State]s yielded after are ignored and will not result in a [Transition].
+  @override
   @mustCallSuper
-  void dispose() {
+  void close() {
     _eventSubject.close();
     _stateSubject.close();
   }
 
+  @Deprecated('Please use add instead.')
+  void dispatch(Event event) => add(event);
```

Comment:
probably wanna keep @mustCallSuper here for the time being as well

Reply:
good catch!